internal/view/live_view.go

```diff
@@ -132,6 +133,7 @@ func (v *LiveView) bindKeys() {
 		tcell.KeyCtrlS:  ui.NewKeyAction("Save", v.saveCmd, false),
 		ui.KeyC:         ui.NewKeyAction("Copy", v.cpCmd, true),
 		ui.KeyF:         ui.NewKeyAction("Toggle FullScreen", v.toggleFullScreenCmd, true),
+		ui.KeyP:         ui.NewKeyAction("Enable/Disable Auto-Refresh", v.pauseCmd, true),
```

Comment:
```suggestion
		ui.KeyP:         ui.NewKeyAction("Toggle Auto-Refresh", v.pauseCmd, true),
```
To match various other places